src/mokka/src/main/java/pl/hycom/mokka/endpoint/FileController.java

```diff
@@ -53,34 +50,20 @@
     @ResponseBody
     public ResponseEntity<FileSystemResource> fetchFile(
             @PathVariable("file-id")
-            String fileId,
+                    String fileId,
             @RequestParam(name = "ext")
-            String extension) {
+                    String extension) throws FileNotFoundException, IllegalArgumentException {
```

Comment:
IllegalArgumentException is unchecked so we do not need to define it on method signature